webpack.tmpl.config.js

```diff
@@ -16,7 +16,28 @@ var config = {
       uglify: true,
       context: require(path.resolve(baseDir, 'colors.json')),
       page: 'colors.html'
-    }
+    },
+    {
+      template: 'src/templates/license.ejs',
+      filename: 'license.html',
+      uglify: true,
+      context: require(path.resolve(baseDir, 'license.json')),
+      page: 'license.html'
+    },
+    {
+      template: 'src/templates/about-pylons.ejs',
+      filename: 'about-pylons.html',
+      uglify: true,
+      context: require(path.resolve(baseDir, 'about-pylons.json')),
+      page: 'about-pylons.html'
+    },
+    {
+      template: 'src/templates/history.ejs',
+      filename: 'history.html',
+      uglify: true,
+      context: require(path.resolve(baseDir, 'history.json')),
+      page: 'history.html'
+    },
```

Comment:
better not for sure, but this template is not rendered by the browser but with outputted with webpack. Make sure people have their editor using the eslint file, or add one your editor support.